src/utils/stations.ts

```diff
@@ -46,7 +46,11 @@ export const getStationsFromShutdown = (stop: string, line: Lines) => {
 };
 
 // This assumes the stops are on the same line
-export const findStopsBetween = (startStation: string, endStation: string, line: Lines) => {
+export const findStopsBetween = (
+  startStation: string | undefined,
+  endStation: string | undefined,
```

Comment:
we can just do `?` here in the type

Reply:
So the advantage of being explicit is that it still requires you pass something into the prop, even if it's undefined, but you can't just forget to include the prop

Reply:
ahh gotcha